src/src/com/microsoft/aad/adal/HttpWebRequest.java

```diff
@@ -144,7 +142,7 @@ public HttpWebResponse send() throws IOException {
             }
             // GET request should read status after getInputStream to make
             // this work for different SDKs
-            final int statusCode = getStatusCode(connection);
+            final int statusCode = connection.getResponseCode();
```

Comment:
? what do you mean by "not handle", do you think we should crash?
this method throws IOException in multiple places, we handle them all